tools/pyteomics/mztab2tsv.xml

```diff
@@ -0,0 +1,98 @@
+<tool id="mztab2tsv" name="pyteomics: convert mztab to tabular" version="@TOOL_VERSION@" profile="20.01" license="TODO_license_in_spdx_format">
+    <description></description>
+    <macros>
+        <token name="@TOOL_VERSION@">4.4.1</token>
+        <xml name="output" token_type="" token_label="">
+            <data name="out_@TYPE@" format="tabular" from_work_dir="@TYPE@.tsv" label="${tool.name} on ${on_string}: @LABEL@">
+                <filter>"@TYPE@" in out_select</filter>
+            </data>
+        </xml>
+    </macros>
+    <xrefs>
+        <xref type="bio.tools">pyteomics</xref>
+    </xrefs>
+    <edam_topics>
+        <edam_topic>topic_0121</edam_topic><!-- proteomics -->
+        <edam_topic>topic_3520</edam_topic><!-- proteomics experiment-->
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_3434</edam_operation><!--	Convert a data set from one form to another -->
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">pyteomics</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ## make sure that selected outputs exist even if the tool does not generate them
+    #for o in $out_select
+        touch "$o".tsv &&
+    #end for
+    '$__tool_directory__/mztab_reader.py' --path_in '$path_in'
+    ]]></command>
+    <inputs>
+        <param argument="--path_in" type="data" format="mztab,mztab2" label="mztab or mztab2 data set" help="" />
+        <param name="out_select" type="select" label="Select desired tables" multiple="true" help="">
+            <option value="mtd" selected="true">Metadata (v1,v2)</option>
+            <option value="prt">Protein table (v1)</option>
+            <option value="pep">Peptide table (v1)</option>
+            <option value="psm">Peptide spectrum match table (v1)</option>
+            <option value="sml">Small molecule table (v1,v2)</option>
+            <option value="smf">Small molecule feature table (v2)</option>
+            <option value="sme">Small molecule evidence table (v2)</option>
+        </param>
+    </inputs>
+    <outputs>
+        <expand macro="output" type="mtd" label="Metadata"/>
+        <expand macro="output" type="prt" label="Proteins"/>
+        <expand macro="output" type="pep" label="Peptides"/>
+        <expand macro="output" type="psm" label="Peptide spectrum matches"/>
+        <expand macro="output" type="sml" label="Small molecules"/>
```

Comment:
are all filetypes available in Galaxy?

Reply:
these are all tabular files .. see the macro?